docs/build/collator/collator-related-operation-guide.md

```diff
@@ -6,45 +6,45 @@
    RPC calls to HTTP endpoints using the `author_rotateKeys` method in your
    terminal.
 
-```
-curl http://127.0.0.1:9933 -H \
-"Content-Type:application/json;charset=utf-8" -d \
-  '{
-    "jsonrpc": "2.0",
-    "id":1,
-    "method": "author_rotateKeys",
-    "params": []
-  }'
-```
-
-2. Mapping Author IDs and setting session keys:
+   ```
+   curl http://127.0.0.1:9933 -H \
+   "Content-Type:application/json;charset=utf-8" -d \
+     '{
+       "jsonrpc": "2.0",
+       "id":1,
+       "method": "author_rotateKeys",
+       "params": []
+     }'
+   ```
+
+1. Mapping Author IDs and setting session keys:
 
    1. Head to
       [Polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fbsr.zeitgeist.pm#/accounts)
 
-   2. Choose `Developer`-`Extrinsics`
+   1. Choose `Developer`-`Extrinsics`
 
-   3. Select the account you want to be associated
+   1. Select the account you want to be associated
 
-   4. Choose `authorMapping` and `addAssociation(authorId)`
+   1. Choose `authorMapping` and `addAssociation(authorId)`
 
-   5. Fill your author Id and submit transaction
+   1. Fill your author Id and submit transaction
 
```

Comment:
I am finding something strange here, not sure if its on my end.

All the steps appear to be numbered as 1 here, but on checking the file they seem to be numbered in ascending order. The preview looks as required.

Reply:
Using `1.` repeatedly is how you get the markdown renderer to automatically enumerate.